### PR TITLE
Fixed undefined constant error when using PantherTestCaseTrait

### LINKED ISSUE #384: Undefined class constant 'CHROME'
Hi,

Symfony 4.4 fails when using `PantherTestCaseTrait`:

> Undefined class constant 'CHROME'

I see https://github.com/symfony/panther/issues/317 and that it is merged but the error is still happening (starting with 0.7.0 and to the latest).
I created a reproducible example https://github.com/aurelijusrozenas/panther-test

The test code taken from example and is minimal: https://github.com/aurelijusrozenas/panther-test/blob/main/tests/DefaultTest.php

Any ideas? I would say it's a bug but how come no one has reported yet? :)

### PR DESCRIPTION
**Pull request by nusje2000**
Fixes #384

### CODE REVIEW

**Review by OskarStark — APPROVED** [review]

**Comment by dunglas** [comment]
I prefer to wait for native support of enums that will land in PHP 8.1 instead of doing our own thing.

File: src/Browser.php [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Symfony\Component\Panther;
+
+final class Browser
```
Comment (jderusse) [inline src/Browser.php:16]: `Browers` for consistency with sulfony's `Events`?

Comment (nusje2000) [reply src/Browser.php:16]: I agree, I will fix this if we indeed want this PR to go to master.

**Comment by nusje2000** [comment]
@dunglas the transition from this class to an enum is probably not that complex and since symfony now supports 7.1, we would probably talking about years in the future for this to be possible. The current implementation is very simple and is only for moving the responsibility from the TestCase to a seperate class.

Another option imo would be to move the constants to the trait instead of the PantherTestCase, this would make more sense since now the trait does not have a dependency on its parent.

What do you think?

**Comment by nusje2000** [comment]
@dunglas  would you mind to have a look at this PR again, I changed the scope to only fix the bug without introducing a new enum.

**Review by dunglas — APPROVED** [review]

**Comment by nusje2000** [comment]
@dunglas can this PR get merged?

**Comment by dunglas** [comment]
Thank you @nusje2000!

**Pull request merged**